xml/release-notes.xml

```diff
@@ -398,6 +410,87 @@
       <package>kdesdk-scripts</package>.
      </para>
     </listitem>
+    <listitem>
+     <para>
+      <package>kdeuser</package>: Replaced by
+      <package>kde-user-manager</package>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <package>kile5</package>: Replaced by
+      <package>kile</package>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <package>kpassgen</package>
+      <package>kpassx</package>: Both replaced by
+      <package>kpassxc</package>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <package>kaccessible</package>
+      <package>kepas</package>
+      <package>konsole4</package>
+      <package>klinkstatus</package>
+      <package>kppp</package>
+      <package>kremotecontrol</package>
+      <package>kvpnc</package>
+      <package>kvkbd</package>: Removed as package is no longer
+        released nor maintained by the KDE community.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <package>fate</package>: Removed as it uses insecure KDE4 and Qt4 libraries and
+       features.opensuse.org is no longer used for feature requests.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <package>gstreamer-plugins-qt</package>
+      <package>gstreamer-plugins-qt</package>
+      <package>ktp-call-ui</package>: Removed as package
+       is unmaintained and no longer builds.
+       Package ktp-call-ui depends on gstreamer-plugins-qt.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <package>H2rename</package>: Removed as package was
+       already removed from Factory.
```

Comment:
Better say that this is "not maintained" instead.

"Also removed from factory" is not useful reasoning for users.